stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -111,11 +118,13 @@ public String run(String[] args) throws Exception {
       jc.parse(args);
       if (mBaseParameters.mHelp) {
         jc.usage();
+        System.out.println(getBenchDescription());
         System.exit(0);
       }
     } catch (Exception e) {
       LOG.error("Failed to parse command: ", e);
```

Comment:
Since the command fails to parse I think we should print some information about the error in the stdout too. Maybe we can add below
```
System.out.format("Failed to parse command: %s%n", e.getMessage());
```
And print the full error just `LOG.error("Failed to parse command: ", e); ` to the log.

I'm not sure if `e.getMessage()` is helpful enough. Please do a few tests and feel free to improve that part too. Thx!

Reply:
It is taken care of here: https://github.com/Alluxio/alluxio/blob/b1db0716971c094300d4d4157746630f4db9138e/stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java#L78 It's printing the stack trace.